nwb_conversion_tools/datainterfaces/ceddatainterface.py

```diff
@@ -13,7 +13,20 @@ class CEDRecordingInterface(BaseRecordingExtractorInterface):
     @classmethod
     def get_source_schema(cls):
         """Compile input schema for the RecordingExtractor."""
-        return get_schema_from_method_signature(
+        source_schema = get_schema_from_method_signature(
             class_method=cls.RX.__init__,
-            exclude=['smrx_ch_inds']
+            exclude=['smrx_channel_ids']
         )
+        source_schema.update(additionalProperties=True)
+        source_schema['properties'].update(
+            file_path=dict(
+                type=source_schema['properties']['file_path']['type'],
+                format="file",
+                description="path to data file"
+            )
+        )
```

Comment:
```suggestion
        source_schema['properties']['file_path'].update(
            format="file",
            description="path to data file"
        )
```

would something like this work?